packages/components/src/components/button-link/button-link.jsx

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-const ButtonLink = ({ children, className, to, onClick }) => (
+const ButtonLink = ({ children, className, to, onClick, size = 'medium' }) => (
     <Link
-        className={classNames('dc-btn dc-btn--primary dc-btn__medium', className, 'effect')}
+        className={classNames('dc-btn dc-btn--primary', className, 'effect', `dc-btn__${size}`)}
```

Comment:
this should be a modifier, shouldn't it?

Reply:
since our buttons are styled this way, I'm following the same